API/Controllers/EmbedController.cs

```diff
@@ -63,7 +64,11 @@ public EmbedController(IEmbedService embedService, IMapper mapper, IProjectServi
         /// Gets all embedded projects.
         /// </summary>
         /// <returns>A list of embedded projects Resource Result.</returns>
+        /// <response code="200">Returns a list of embedded project resource results.</response>
```

Comment:
```suggestion
        /// <response code="200">This endpoint returns a list with embedded projects.</response>
```

Reply:
In addition to the comments of @jcoenen96:

- Never use a period if the comment is not a proper sentence.

- Instead of only adding periods to the comments, all comments should be proper sentences with correct interpunction.

I don't know if this is in the coding guidelines. If it is not, I agree with @jcoenen96 that it should be added.
See the example above.

Reply:
So what would you suggest doing in these cases?
`// <param name="mapper">The mapper.</param>`

I've seen cases where there is a dot and I've seen cases where there is no dot. So should I add dots to every place or should I remove all the dots here so we get some better consistency? Thank you 😄

Reply:
And same question applies to this
`    /// <summary>
    /// This controller handles the CRUD roles.
    /// </summary>`

Reply:
> So what would you suggest doing in these cases?
> `// <param name="mapper">The mapper.</param>`
> 
> I've seen cases where there is a dot and I've seen cases where there is no dot. So should I add dots to every place or should I remove all the dots here so we get some better consistency? Thank you 😄

I would write something like "The mapper which is used to convert the ... resource to the ... class." or something like that. Explain what it is / does.
Only use periods if the comment itself is a proper sentence.




> And same question applies to this
> ` /// <summary> /// This controller handles the CRUD roles. /// </summary>`

The same thing here. Describe the functionality of the method or class. For example "This class is responsible for handling HTTP requests that are related to the ... class. For example creating, retrieving, updating or deleting ...".